generators/php/text.js

```diff
@@ -153,14 +153,12 @@ Blockly.PHP['text_getSubstring'] = function(block) {
   // Get substring.
   var where1 = block.getFieldValue('WHERE1');
   var where2 = block.getFieldValue('WHERE2');
+  var text = Blockly.PHP.valueToCode(block, 'STRING',
+      Blockly.PHP.ORDER_NONE) || '\'\'';
   if (where1 == 'FIRST' && where2 == 'LAST') {
-    var text = Blockly.PHP.valueToCode(block, 'STRING',
-        Blockly.PHP.ORDER_NONE) || '\'\'';
     var code = text;
     return [code, Blockly.PHP.ORDER_NONE];
   } else {
-    var text = Blockly.PHP.valueToCode(block, 'STRING',
-        Blockly.PHP.ORDER_NONE) || '\'\'';
```

Comment:
If you delete text here do you need to delete below in `var code = functionName + '(' + text + ', \''`

Reply:
I didn't just remove it, I moved it to earlier line, since logic in both if and else both set it to the same value (and use the same order).